Index: src/main/java/ru/madbrains/lesson2/Calculator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.madbrains.lesson2;public class Calculator {\n}\n
===================================================================
diff --git a/src/main/java/ru/madbrains/lesson2/Calculator.java b/src/main/java/ru/madbrains/lesson2/Calculator.java
--- a/src/main/java/ru/madbrains/lesson2/Calculator.java	
+++ b/src/main/java/ru/madbrains/lesson2/Calculator.java	
@@ -1,2 +1,9 @@
-package ru.madbrains.lesson2;public class Calculator {
+package ru.madbrains.lesson2;
+
+
+public class Calculator {
+
+    public static void main(String[] args) {
+
+    }
 }
